examples/all-clusters-app/esp32/main/DeviceWithDisplay.cpp

```diff
@@ -196,6 +196,36 @@ class EditAttributeListModel : public TouchesMatterStackModel
                 ESP_LOGI(TAG, "SystemMode changed to : %d", n);
                 app::Clusters::Thermostat::Attributes::OccupiedHeatingSetpoint::Set(1, n);
             }
+            else if (name == "Current Lift")
+            {
+                // update the current lift here for hardcoded endpoint 1
+                ESP_LOGI(TAG, "Current position lift percent 100ths changed to : %d", n);
```

Comment:
Shouldn't that be `n*100` for the value being logged?  Same for the tilt case.

Reply:
It should what is the base type of 'n'

Reply:
changed to `n*100`